pom.xml

```diff
@@ -30,8 +32,7 @@
                                 <groupId>org.apache.maven.plugins</groupId>
                                 <artifactId>maven-compiler-plugin</artifactId>
                                 <configuration>
```

Comment:
you should not need the configuration tag once you update the parent. you can get rid of it